8165 - Updated colors in colorpicker

max-hizon opened the pull request:
**Explain the _details_ for making this change. What existing problem does the pull request solve?**
This pull request will update the color palette used in colorpicker

**Related github/jira issue (required)**:
Closes: https://github.com/infor-design/enterprise/issues/8165

**Steps necessary to review your pull request (required)**:
1. Go to http://localhost:4000/components/colorpicker/example-index.html
2. Click the color picker drop down
3. See that the color options are now updated to match current color palette. Use https://design.infor.com/code/ids-enterprise/latest/demo/components/colors/example-index?theme=uplift&variant=light as reference

**Included in this Pull Request**:
- [ ] An e2e or functional test for the bug or feature.
- [x] A note to the change log.

<!-- After submitting your PR, please check back to make sure tests pass on the github actions. -->

tmcconechy reviewed (changes requested):
Looks good on the color picker changes but looks like the CHANGELOG and maybe contextual action panel are deleted/or changed by accident?

max-hizon commented:
> Looks good on the color picker changes but looks like the CHANGELOG and maybe contextual action panel are deleted/or changed by accident?

I think I must've accidentally worked on a different branch while using this one. Will fix.

max-hizon commented:
@tmcconechy After reviewing the code it seems for some reason when I edited CHANGELOG.md and contextual action panel, the edit made it so it erased its contents and replaced it with the same code along with the changes I made. For contextual action panel, it seems to have just deleted the code and replaced it with the same.

yohannahbautista commented on docs/CHANGELOG.md:
```diff
@@ -8,14 +8,14 @@
 - `[Modal]` Adjusted modal title spacing to avoid icon from cropping. ([#8031](https://github.com/infor-design/enterprise/issues/8031))
 - `[Page-Patterns]` Fixed background on hovered selected tab. ([#8088](https://github.com/infor-design/enterprise/issues/8088))
 - `[Popupmenu]` Added fix for icon size in new. ([#8175](https://github.com/infor-design/enterprise/issues/8175))
-- `[Tabs-Module]` Removed padding in embedded mode. ([#8060](https://github.com/infor-design/enterprise/issues/8060))
```
Unless this was removed to fix sorting or it's a duplicate, please undo this

tmcconechy reviewed (approved):
Thanks @max-hizon no problem. I do that too sometimes. Just flagging it. (one tip is i like to review my own PR when i first put it in).

Cheers 👍🏻 Nice Work here

jbrcna reviewed (approved):
nice work @max-hizon . are you a dev now?
<img width="473" alt="image" src="https://github.com/infor-design/enterprise/assets/56722184/b55686a7-33ab-44a1-b5f2-085268f8da32">

glenlieorillo reviewed (approved):
<img width="750" alt="image" src="https://github.com/infor-design/enterprise/assets/102151352/6f1a639a-a4eb-444f-83cb-6c3f9ca0561c">

<img width="750" alt="image" src="https://github.com/infor-design/enterprise/assets/102151352/5893d468-ef0e-4957-abf1-1039d03bcbdf">
